ui-app/client/src/components/FormStorageEditor/components/formCommons.ts

```diff
@@ -0,0 +1,586 @@
+			maxChars: 10,
+			validation: {
+				STRING_LENGTH: {
+					uuid: '',
+					order: 0,
+					type: 'STRING_LENGTH',
+					minLength: 10,
+					maxLength: 10,
+					message: 'not valid length',
+				},
+			},
+			schema: {
+				type: ['STRING'],
+				minLength: 10,
+				maxLength: 10,
+			},
+			editorType: 'PhoneEditor',
+		},
+	],
+	[
+		'textBox',
+		{
+			key: 'textBox',
+			uuid: '',
+			order: undefined,
+			label: 'Text box',
+			placeholder: 'Type something',
+			inputType: 'text',
+			numberType: '',
+			validation: {
+				STRING_LENGTH: {
+					uuid: '',
+					order: 0,
+					type: 'STRING_LENGTH',
+				},
+			},
+			schema: {
+				type: ['STRING'],
+			},
+			editorType: 'TextBoxEditor',
+		},
+	],
+	[
+		'textArea',
+		{
+			key: 'textArea',
+			uuid: '',
+			order: undefined,
+			label: 'Text area',
+			placeholder: 'Type description',
+			maxChars: 2000,
+			inputType: 'text',
+			validation: {
+				STRING_LENGTH: {
+					uuid: '',
+					order: 0,
+					type: 'STRING_LENGTH',
+					minLength: 0,
+					maxLength: 2000,
+					message: 'not valid length',
+				},
+			},
+			schema: {
+				type: ['STRING'],
+				minLength: 0,
+				maxLength: 2000,
+			},
+			editorType: 'TextAreaEditor',
+		},
+	],
+	[
+		'dropdown',
+		{
+			key: 'dropdown',
+			uuid: '',
+			order: undefined,
+			label: 'Drop down',
+			placeholder: 'Select an option',
+			isMultiSelect: false,
+			validation: {},
+			optionList: [
+				{
+					key: 'option 1',
+					value: 'Option 1',
+				},
+				{
+					key: 'option 2',
+					value: 'Option 2',
+				},
+			],
+
+			schema: {
+				type: ['STRING'],
+				enums: ['option1', 'option2'],
+			},
+
+			editorType: 'DropdownEditor',
+		},
+	],
+	[
+		'radioButton',
+		{
+			key: 'radioButton',
+			uuid: '',
+			order: undefined,
+			label: 'Radio Button',
+			validation: {},
+			optionList: [
+				{
+					key: 'option 1',
+					value: 'Option 1',
+				},
+				{
+					key: 'option 2',
+					value: 'Option 2',
+				},
+			],
+
+			schema: {
+				type: ['STRING'],
+				enums: ['option1', 'option2'],
+			},
+			editorType: 'RadioButtonEditor',
+		},
+	],
+	[
+		'checkBox',
+		{
+			key: 'checkBox',
+			uuid: '',
+			order: undefined,
+			label: 'Check box',
+			isMultiSelect: true,
+			validation: {},
+			optionList: [
+				{
+					key: 'option 1',
+					value: 'Option 1',
+				},
+				{
+					key: 'option 2',
+					value: 'Option 2',
+				},
+			],
+			schema: {
+				type: ['ARRAY'],
+				enums: ['option1', 'option2'],
+				items: {
+					type: ['STRING'],
+				},
+			},
+			editorType: 'CheckBoxEditor',
+		},
+	],
+]);
+
+export interface FormCompValidation {
+	uuid: string;
+	type: string;
+	order: number | undefined;
+	message?: string;
+	pattern?: string;
+	ignoreCase?: boolean;
+	minLength?: number | undefined;
+	maxLength?: number | undefined;
+	minValue?: number | undefined;
+	maxValue?: number | undefined;
+}
+
+export const compValidationMap = new Map<string, FormCompValidation>([
```

Comment:
Sorry forgot to infrom you, usually all exported constants should follow the ALL_CAPS_WITH_UNDERSCORE_SPERATOR naming convention. COMP_VALIDATION_MAP. Change the others.